core/unit_test/cuda/TestCuda_InterOp_Streams.cpp

```diff
@@ -156,35 +63,35 @@ TEST(cuda, raw_cuda_streams) {
 
     Kokkos::parallel_for("Test::cuda::raw_cuda_stream::Range",
                          Kokkos::RangePolicy<Kokkos::Cuda>(cuda0, 0, 100),
-                         FunctorRange(v));
+                         FunctorRange<Kokkos::CudaSpace>(v));
```

Comment:
Any reason to prefer this to deducing execution and memory spaces from the TEST_EXECSPACE macro?

Reply:
No. Does it matter?

Reply:
To me yes.

Reply:
Why though?

Reply:
Because it hurts maintainability.
If a brave soul decide to work on generating the test .cpp files from CMake he will have to diff these guys to understand what is going on.  I am open to defer doing that if you add a comment as a  weak attempt to prevent someone else to edit one file but not the other.
I do see that it is not as simple as the definition of these two spaces because of things like https://github.com/kokkos/kokkos/blob/f376c99423f4f794109d45b1d96bdd8abd91272d/core/unit_test/cuda/TestCuda_InterOp_Streams.cpp#L100-L105 and I understand why you would be rightfully reluctant to deal with it.

TLDR add a comment where you think is appropriate and you've got my approval.

Reply:
> I understand why you are reluctant to deal with it

I am not reluctant. I just wanted to know the reason why you wanted this change.